src/Semoda/Semoda/Models/AppSettingsModel.cs

```diff
@@ -0,0 +1,61 @@
+﻿using System.IO;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Linq;
+using System.Runtime.CompilerServices;
+using System.Text;
+using System.Text.Json;
+using System.Threading.Tasks;
+
+namespace Semoda.Models
+{
+    /// <summary>
+    /// Model for the settings in the application.
+    /// All settings have default values - if no settings file is loaded, these are applied.
+    /// A settings file is first created when the settings are saved from the settings view.
+    /// Next time the app launches, those new settings will be loaded at startup.
+    /// </summary>
+    public class AppSettingsModel : INotifyPropertyChanged
```

Comment:
Instead of using the INotifyPropertyChanged interface, the ObservableObject of the MVVM Toolkit should be used.